Readme.md

```diff
@@ -157,18 +157,34 @@ A detailed explaination of these settings can be found on the [rclone website](h
 This section needs to be done. Just a few command snippets:
 
 * Is rclone running?
-```
-# logs
-journalctl -t rclone
+  ```
+  # logs
+  journalctl -t rclone
 
-# systemd status
-systemctl status edge-cacher-<share name>
-```
+  # systemd status
+  systemctl status edge-cacher-<share name>
+  ```
 
 * Is samba running?
-```
-sudo systemctl status smbd.service
-```
+  ```
+  sudo systemctl status smbd.service
+  ```
+
+* Is the certificate trusted?
+  If you are using a self signed certificate, don't forget to add it to your trusted certifcates
+  You can check this by running:
+  ```
+  sudo systemctl status edge-cacher-<share name>
+
+  # Output
+  # IO error: couldn't list files: Propfind "<your nextcloud url>": tls: failed to verify certificate: x509: certificate signed by unknown authority
+  ```
+
+  You can fix this error by add your certificate to `/usr/local/share/ca-certificates/` (Ubuntu) and then run:
+  ```
+  sudo update-ca-certificates
+  ```
```

Comment:
Thanks, nice addition. Didn't run into this, as I was using Let's encrypt certs on my side.